test/benchmark/lmbench-lmdd/config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "alert_threshold": "125%",
+    "alert_tool": "customBiggerIsBetter",
+    "search_pattern": "lmdd result:",
+    "result_index": "7",
+    "description": "The bandwidth of I/O on a single processor."
```

Comment:
This description is vague. What do you mean exactly with "I/O"? There could be many ways of doing I/O.